### PR TITLE
Improve Login page styles

### PR DESCRIPTION
**Pull request by marco-souza**
Mockups: https://www.figma.com/file/STYpmAiDlsA0jjaZGS4JIa/cantoDaRua?node-id=0%3A1

I've also added a dummy password validation

### CODE REVIEW

File: components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    label: {
+      color: palette.primary.main,
+    },
+  }),
+);
+
+function Button(props: Props) {
+  const classes = useStyles();
+  // const buttonStyles = useVariantStyles(variant);
```
Comment (eduhdm) [inline components/Button.tsx]: Please, remove this comment

File: components/LoginPage.tsx [hunk 2]
```diff
@@ -1,40 +1,93 @@
...
-      margin: '10rem auto',
-      width: '20rem',
-      minWidth: `5rem`,
+      margin: '5rem auto',
+      maxWidth: '20rem',
       padding: `1rem`,
       borderRadius: `0.5rem`,
-      border: 'solid 2px white',
       textAlign: `center`,
+      display: `flex`,
```
Comment (eduhdm) [inline components/LoginPage.tsx]: Is there a specific reason to use template strings on regular strings? Could you please change to use a `'`?

File: components/LoginPage.tsx [hunk 3]
```diff
@@ -1,40 +1,93 @@
...
     googleButton: {
-      backgroundColor: 'red',
-    },
-    logoImage: {
       width: '100%',
     },
+    googleIcon: {
+      margin: '0 0.5rem',
+    },
+    helpingHandIcon: {
```
Comment (eduhdm) [inline components/LoginPage.tsx]: Where is this class being used? If it's not used anywhere we can remove it

File: components/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+interface Props extends ButtonProps {
+  children: ReactNode;
+}
+
+const useStyles = makeStyles(({ palette }: Theme) =>
+  createStyles({
+    outlined: {
+      borderColor: palette.primary.main,
+      // padding: '10px 0',
```
Comment (eduhdm) [inline components/Button.tsx]: Please, remove this comment

**Comment by marco-souza** [comment]
Sorry Mr. @eduhdm, it won't happen again! :)

**Review by eduhdm — APPROVED** [review]

**Pull request merged**